scripts/transforms/custom-value-css-box-shadow.transform.js

```diff
@@ -17,6 +19,17 @@ module.exports = {
       return 'none'
     }
 
+    if (isArray(value)) {
+      return value.map(val => {

```

Comment:
Please do not use shortened names of the variables.
Since boxShadows contain shadow definitions, I'd go with `shadowDefinition` or `definition`, name instead of `val`.